sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -142,6 +142,102 @@ def hpat_pandas_series_getitem_idx_series_impl(self, idx):
     raise TypingError('{} The index must be an Integer, Slice or a pandas.series. Given: {}'.format(_func_name, idx))
 
 
+@sdc_overload(operator.setitem)
+def hpat_pandas_series_setitem(self, idx, value):
+    """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+    Pandas API: pandas.Series.set
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series_setitem_int.py
+       :language: python
+       :lines: 27-
+       :caption: Setting Pandas Series elements
+       :name: ex_series_setitem
+
+    .. code-block:: console
+
+        > python ./series_setitem_int.py
+
+            0    0
+            1    4
+            2    3
+            3    2
+            4    1
+            dtype: int64
+
+        > python ./series_setitem_slice.py
+
+            0    5
+            1    4
+            2    0
+            3    0
+            4    0
+            dtype: int64
+
+        > python ./series_setitem_series.py
+
+            0    5
+            1    0
+            2    3
+            3    0
+            4    1
+            dtype: int64
+
+    Intel Scalable Dataframe Compiler Developer Guide
+    *************************************************
+     Pandas Series operator :attr:`pandas.Series.set` implementation
+
+    Test: python -m sdc.runtests sdc.tests.test_series.TestSeries.test_series_setitem_unsupported
```

Comment:
Please use a correct line to run the tests for setitem, i.e.
`python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_series_setitem*`